src/components/homepage-features.tsx

```diff
@@ -103,7 +102,32 @@ export default function HomepageFeatures(): JSX.Element {
       <div className={clsx('container--small', styles.footerCta)}>
         <h2>Ready to get started with Sequelize?</h2>
         <p>
-          Transactions, migrations, strong typing, JSON querying, and more.
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/transactions/"
+          >Transactions
+          </Link>,&nbsp;
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/migrations/"
+          >migrations
+          </Link>,&nbsp;
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/typescript/"
+          >strong typing
+          </Link>,&nbsp;
```

Comment:
Don't use non breaking spaces for these, use a regular space instead:

```suggestion
          </Link>,{' '}
```